Comparison Engine/Comparison Engine/Base Classes/Bar.cs

```diff
@@ -20,16 +20,11 @@ public Bar(int barID, string barName, string barLocation)
             this.barLocation = barLocation;
         }
 
-        //Changes this bar object's name
-        public void EditBarName(string newName)
+        //Changes properties of this Bar Object
+        public void EditBar(BarStruct barStruct)
         {
-            barName = newName;
-        }
-
-        //Changes this bar object's location
-        public void EditBarLocation(string newLocation)
-        {
-            barLocation = newLocation;
+            if (barStruct.barName != null) this.barName = barStruct.barName;
+            if (barStruct.barLocation != null) this.barLocation = barStruct.barLocation;
         }
```

Comment:
I don't know if we should write ifs without curly brackets stylistically

Reply:
Fair point